### PR TITLE
Adding more items to navbar menu and make hamberger menu for small devices

### PR DESCRIPTION
**Pull request by DDHLong**

### CODE REVIEW

**Comment by McSinyx** [comment]
You probably should remove the unused class names in HTML tags IMHO.  When you think it's ready, please also provide screenshots of the affected pages.

File: src/acanban/templates/base.html [hunk 1]
```diff
@@ -18,46 +18,56 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8">
+    <title>Acanban</title>
     <link rel="stylesheet" type="text/css" href="/static/style.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Public+Sans&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;600;700;800&display=swap" rel="stylesheet">
-    <title>{{title}}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
```
Comment (McSinyx) [inline src/acanban/templates/base.html:27]: What is this for?  Awesome works with just CSS.

File: src/acanban/static/script.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector(".menu-icon span");
+    const searchBtn = document.querySelector(".search-icon");
```
Comment (McSinyx) [inline src/acanban/static/script.js:2]: Notice the indentation issue here.

I suppose [it's possible w/o JS](https://code-boxx.com/simple-responsive-pure-css-hamburger-menu/), although I'm not too much against a 25 LoC JS, long as the page works without it and it is properly licensed.

File: src/acanban/templates/base.html [hunk 3]
```diff
@@ -18,46 +18,56 @@
...
   </head>
   <body>
     {% if error %}
     <p>error: {{ error }}</p>
     {% endif %}
```
Comment (McSinyx) [inline src/acanban/templates/base.html:32]: This is not styled and is not in the right place, i.e. we missed it last iteration.

Comment (Huy-Ngo) [reply src/acanban/templates/base.html:32]: Where do you want it? In a popup?

File: src/acanban/static/script.js (hunk 2 above)
Comment (Huy-Ngo) [reply src/acanban/static/script.js:2]: You can see [GNU guidelines](https://www.gnu.org/software/librejs/free-your-javascript.html)

File: src/acanban/templates/base.html (hunk 3 above)
Comment (McSinyx) [reply src/acanban/templates/base.html:32]: Nope, as something just before the form (I think this should be page specific too).

Comment (Huy-Ngo) [reply src/acanban/templates/base.html:32]: Then isn't it already at the right place.

Comment (McSinyx) [reply src/acanban/templates/base.html:32]: Nope, this is above the navbar, which brings some weird experience.

**Comment by McSinyx** [comment]
Superseded by GH-73, thank you for your patience though.

**Pull request closed without merging**